CHANGELOG.md

```diff
@@ -23,6 +23,16 @@ WYMeditor.
 * [#648](https://github.com/wymeditor/wymeditor/pull/647),
   [#641](https://github.com/wymeditor/wymeditor/pull/641)
   Object resizing (images, primarily) in Firefox and IE has been disabled.
+* [#639](https://github.com/wymeditor/wymeditor/pull/639)
+  * The `jQueryPath` option was removed.
+  * The `wym._index` editor instance property is removed.
+  * Removed `wym.wrap` and `wym.unwrap` editor methods.
+  * Dialogs cannot be opened using the `exec` method. Use the `dialog` method,
+    instead.
+  * Changes in `wym.dialog` method. Existing custom dialogs should still work,
+    but it is recommended that existing custom dialogs be rewritten using the
+    new `wym.dialog` API (see API documentation) or at least that they be
+    tested.
```

Comment:
It seems obvious to me that removal of public API must be mentioned in backward incompatible changes.